L1Trigger/L1THGCalUtilities/python/calo.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+from L1Trigger.L1THGCalUtilities.caloTruthCellsProducer_cfi import caloTruthCellsProducer
+
+def create_calo(process, inputs):
```

Comment:
I think the name "calo" here (and everywhere else) is not very clear. I would use "truth_something".

Reply:
Renamed "calo" to "truth_prod" throughout.